runtime/CMakeLists.txt

```diff
@@ -3,14 +3,21 @@ option(TTMLIR_ENABLE_RUNTIME_TESTS "Enable runtime tests" OFF)
 option(TT_RUNTIME_ENABLE_TTNN "Enable TTNN Runtime" ON)
 option(TT_RUNTIME_ENABLE_TTMETAL "Enable TTMetal Runtime" ON)
 option(TT_RUNTIME_DEBUG "Enable debug tools in runtime" OFF)
+option(TT_RUNTIME_TOGGLE_WORKAROUNDS "Enable toggling workarounds in runtime" OFF)
```

Comment:
Perhaps enable is clearer that toggle here?

Reply:
Yeah I was thinking of using enable, but all workarounds are enabled by default. `TT_RUNTIME_DISABLE_WORKAROUNDS` on the other hand sounds like we're disabling all the workarounds which is sort of misleading as well. Does `TT_RUNTIME_CONFIGURE_WORKAROUNDS` sound better?

Reply:
With the updated comment, I'm on board!